### PR TITLE
add PVXS to the document and separate the 1st and 2nd generation APIs

### PR DESCRIPTION
**Pull request by ttkorhonen**
Update of the PVA libraries page.

### CODE REVIEW

**Comment by ttkorhonen** [comment]
Can (at least one of) you please take a look before I merge. @anjohnson @ralphlange

**Review by minijackson — COMMENTED** [review]
Some very minor nitpicks (feel free to ignore if you disagree)

File: pv-access/overview.rst [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
-
-   pvData <https://docs.epics-controls.org/projects/pvdata-cpp/en/latest>
-   pvAccess <https://docs.epics-controls.org/projects/pvaccess-cpp/en/latest>
-   pva2pva (QSRV / pvAccess Gateway) <https://docs.epics-controls.org/projects/pva2pva/en/latest>
-   Normative Types <https://docs.epics-controls.org/projects/normativetypes-cpp/en/latest>
-   pvaClient <https://docs.epics-controls.org/projects/pvaclient-cpp/en/latest>
-   pvDatabase <https://docs.epics-controls.org/projects/pvdatabase-cpp/en/latest>
+.. tags:: developer, advanced
+
+There are (at the time of writing) two generations of pvAccess C++ implementations available:
```
[the remark this answers is not in the document]
Comment (ttkorhonen) [reply pv-access/overview.rst:6]: I am OK with this, but am a bit unsure what is the correct capitalization...maybe @anjohnson or @ralphlange can pitch in to confirm?

**Comment by ralphlange** [comment]
pvAccess.
https://epics-controls.org/resources-and-support/documents/pvaccess/

File: pv-access/overview.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (ttkorhonen) [reply pv-access/overview.rst:6]: @ralphlange confirmed what I thought...pvAccess seems to be the approved capitalization. I think we should stick to that.

[the remark this answers is not in the document]
Comment (minijackson) [reply pv-access/overview.rst:6]: Oops, my bad then

File: pv-access/overview.rst [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+
+PVXS
+----
+
+PVXS <https://mdavidsaver.github.io/pvxs/> is a re-implementation of the pvAccess protocol support. 
+This module provides a library (libpvxs.so or pvxs.dll) 
+and a set of CLI utilities acting as PVAccess protocol client and/or server.
```
Comment (minijackson) [inline pv-access/overview.rst:13]: Last nitpick, then

```suggestion
and a set of CLI utilities acting as pvAccess protocol client and/or server.
```

Comment (ttkorhonen) [reply pv-access/overview.rst]: Ahh, touché! Thank you for picking the nits 👍

**Pull request merged**